test/images/porter/Makefile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The Kubernetes Authors.
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
These copyright tags should never be updated; only added when a new file is added